google-cloud-bigtable/test/google/cloud/bigtable/admin/v2/bigtable_instance_admin_client_test.rb

```diff
@@ -455,7 +455,7 @@ def updater_proc
               formatted_name,
               display_name,
               type,
-              labels
+              labels.to_h
```

Comment:
Reading this again and I'm thinking this change might not be needed.

Reply:
Do you want to update the PR?

Reply:
I'm going to merge the PR now; if think this isn't needed, I guess we just omit it from the changes to synth.py?

Reply:
Yeah, that's my thinking.